server/scan/device_handling.py

```
@@ -240,6 +240,29 @@ def update_devLastConnection_from_CurrentScan(db):
+    sql = db.sql
+    node_name = str(get_setting_value("SYNC_node_name") or "").strip()
+
+    if not node_name:
+        return
+
+    sql.execute(
+        f"""
+        UPDATE Devices
+        SET devSyncHubNode = ?
+        WHERE COALESCE(LOWER(TRIM(devSyncHubNode)), '') IN {NULL_EQUIVALENTS_SQL}
+        """,
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)